<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageVariables</class>
 <widget class="QWidget" name="WizardPageVariables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Defined parameters and variables</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>7</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ValueRef</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Variability</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Causality</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Initial</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Start value</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonAdd">
        <property name="icon">
         <iconset resource="FMIGenerator.qrc">
          <normaloff>:/gfx/plus.png</normaloff>:/gfx/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButtonRemove">
        <property name="icon">
         <iconset resource="FMIGenerator.qrc">
          <normaloff>:/gfx/minus.png</normaloff>:/gfx/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>236</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="3">
         <widget class="QComboBox" name="comboBoxInitial">
          <item>
           <property name="text">
            <string>exact</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>approx</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>calculated</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Causality:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Variability:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditName"/>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBoxVariability">
          <property name="currentIndex">
           <number>4</number>
          </property>
          <item>
           <property name="text">
            <string>constant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fixed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tunable</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>discrete</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>continuous</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBoxCausality">
          <property name="currentIndex">
           <number>4</number>
          </property>
          <item>
           <property name="text">
            <string>parameter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>calculatedParameter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>input</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>output</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>local</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>independent</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Initial:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxValueRef">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Value reference (-1 for auto):</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Start value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditStart"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxTypeID">
        <item>
         <property name="text">
          <string>Real</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Boolean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>String</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Continuous states</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Number of continuous states in the model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxContinuousStates"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="FMIGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
